backend/test/utils.ml

```diff
@@ -553,6 +553,7 @@ let sample_dvals : (string * dval) list =
   ; ("datastore", DDB "Visitors")
   ; ("string", Dval.dstr_of_string_exn "incredibly this was broken")
   ; ("list", DList [Dval.dint 4])
+  ; ("tuple", DTuple [Dval.dint 4; Dval.dint 5])
```

Comment:
@pbiggar can't get this roundtrip test to pass. I think it has something to do with the `wrap_user_type`. This is the failure

```
[failure] Error roundtrippable: tuple: expecting
(
  4, 5
), got
{
  value: [
    4, 5
  ],
  type: tuple
}.
```

Reply:
Great, we caught a bug!

This needs to be explicitly handled in https://github.com/darklang/dark/blob/main/backend/libexecution/dval.ml#L382